unit_testing/batch_processing/cached_results/mt/mtr_in_wm.csv

```diff
@@ -1,2 +1,2 @@
 Timestamp,SCT Version,Filename,Slice (I->S),VertLevel,Label,Size [vox],MAP(),STD()
-2021-01-25 00:48:42,git-HEAD-42ef57b3a6241a127ed0d7a7eebc909dc46c9d07,/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/sct_example_data/mt/mtr.nii.gz,4:17,2:5,white matter,1866.8310447890826,54.38001541207312,4.851424830774496
+2021-03-16 10:47:36,git-jn/3254-fix-sform-sct_warp_template-9566dce5850797d73d819917f7f49ce74b4a052c,/home/joshua/repos/spinalcordtoolbox/sct_example_data/mt/mtr.nii.gz,4:17,2:5,white matter,980.9416908352437,54.37973987894469,4.85153610548257
```

Comment:
from ~1800 to ~900 that’s a suspiciously large change for « size »

Reply:
Thanks for pointing this out. I'll try to dig a little deeper here. 

(Also relevant: https://github.com/neuropoly/spinalcordtoolbox/issues/3250)

Reply:
if it help: what i would do is open up the WM mask used for this quantification on FSLeyes, before/after the change, and see how they differ

Reply:
Ah! I think this is actually due to a previously fixed bug. Recall: https://github.com/neuropoly/spinalcordtoolbox/issues/3216 and https://github.com/neuropoly/spinalcordtoolbox/pull/3217

Reply:
> Ah! I think this is actually due to a bug that was fixed between #3198 and now. (Issue: #3216 and PR: #3217). mtr_in_wm.csv was generated using the following command (note -method map):

ah yes! good catch!